js/fibo-js.js

```diff
@@ -0,0 +1,111 @@
+let outputPlaceHolder = document.getElementById("fiboOutput");
+let largerPlaceHolder = document.getElementById("numLarger");
+let networkError = document.getElementById("sr-only");
+
+let one_place = document.getElementById("first");
+let second_place = document.getElementById("second");
+let third_place = document.getElementById("third");
+
+let checked = document.getElementById("flexCheckDefault");
+
+function fibo(x) { 
+  i = fiboArray.length;
+  if (fiboArray.length>x) {
+      return(fiboArray[i-1]);
+  }
+      fiboArray.push(fiboArray[i-1]+fiboArray[i-2]);
+      fibo(x);
+}
+let fiboArray = [0,1]
+let i;
+
+window.onload = function() {
+  oldResults();
+};
+
+function compare( a, b ) {
+  if ( a.createdDate < b.createdDate ){
+    return -1;
+  }
+  if ( a.createdDate > b.createdDate ){
+    return 1;
+  }
+  return 0;
+}
+
+serverError=false;
```

Comment:
Where is serverError defined? You always have to define a variable with let/const